tests/unit/block_device_test.cc

```diff
@@ -28,7 +32,7 @@
 
 using namespace seastar;
 
-constexpr off_t min_device_size = 10 << 20; // 10 MiB
+constexpr off_t min_device_size = 16 << 20; // 16 MiB
```

Comment:
we have KB, MB and GB defined in `seastar/core/units.hh`, feel free to use them instead of manual shifts:
`min_device_size = 16*MB`